rekor/rekor.go

```diff
@@ -302,14 +302,14 @@ func ParseEnvelopeFromEntry(entry *models.LogEntryAnon) (dsse.Envelope, error) {
 		_, ok := verifier.(*cryptoutil.X509Verifier)
 		if ok {
 			envSig.Certificate = sig.PublicKey
-			for _, intermediate := range sig.Intermediates {
```

Comment:
I could not find any reference to Intermediate signatures in the rekor signature object so I removed it for now. Happy to re add this if it exists somewhere else.